src/views/design/ScriptDetail/EditAction/index.tsx

```diff
@@ -106,6 +107,7 @@ function EditAction({
     const [condition, setCondition] = useState(action.condition);
     const actionTypes = getAsyncActionTypes(state).data || [];
     const matchingActionType = actionTypes.find((item) => action.type === item.type);
+    const orderedConstantParameters = orderConstantParameters(matchingActionType.parameters, matchingActionType);
```

Comment:
can't you order the parameters when the script (action parameters) are fetched from the backend? cfr. components and connections?